src/main/kotlin/no/nav/syfo/behandlendeenhet/database/NavUtlandQuery.kt

```diff
@@ -0,0 +1,75 @@
+package no.nav.syfo.behandlendeenhet.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.behandlendeenhet.database.domain.PNavUtland
+import no.nav.syfo.domain.PersonIdentNumber
+import java.sql.*
+import java.time.OffsetDateTime
+import java.util.*
+
+const val queryUpdateNavUtland =
+    """
+        INSERT INTO NAV_UTLAND (
+            id,
+            uuid,
+            personident,
+            is_nav_utland,
+            created_at,
+            updated_at
+            ) VALUES (DEFAULT, ?, ?, ?, ?, ?)
+            ON CONFLICT (personident)
+            DO
+                UPDATE SET is_nav_utland = EXCLUDED.is_nav_utland, updated_at = EXCLUDED.updated_at
+            RETURNING
+            id,
+            uuid,
+            personident,
+            is_nav_utland,
+            created_at,
+            updated_at;
+    """
+
+fun DatabaseInterface.updateNavUtland(
+    personIdentNumber: PersonIdentNumber,
+    isNavUtland: Boolean
+): PNavUtland? {
```

Comment:
Trenger denne å returnere noe? 🤔

Reply:
Egentlig ikke, men jeg synes det er fint å kunne returnere en respons når det gjøres en POST i et API. Hva tenker dere?

Reply:
Ser det er anbefalt, men vet ikke om det treffer casen vår helt: https://stackoverflow.com/questions/19199872/best-practice-for-restful-post-response
- Noen mener man skal gi en 201 created, men det tror jeg ikke vi skal bry oss om (og vi kan jo også gjøre en update så det blir bare surr)
- Å gi responsen er best practice, men det er mest for å slippe å GETe samme data rett etterpå. Det kommer vi jo ikke til å gjøre direkte (vi kommer jo til å spørre om behandlende enhet), så det er kanskje ikke et gyldig argument i vårt tilfelle.